client/src/banners/active-banner.tsx

```diff
@@ -64,47 +61,21 @@ function Banner(props: BannerProps) {
   );
 }
 
-// function CommonSurveyBanner({ onDismissed }: { onDismissed: () => void }) {
-//   const ga = useGA();
-
-//   return (
-//     <Banner
-//       id={COMMON_SURVEY_ID}
-//       title={"Learning web development survey"}
-//       copy={
-//         "Help us understand how to make MDN better for beginners (5 minute survey)"
-//       }
-//       cta={"Take the survey"}
-//       url="https://www.surveygizmo.com/s3/6175365/59cfad9c04cf"
-//       newWindow
-//       onDismissed={onDismissed}
-//       onCTAClick={() => {
-//         ga("send", {
-//           hitType: "event",
-//           eventCategory: CATEGORY_LEARNING_SURVEY,
-//           eventAction: "CTA clicked",
-//           eventLabel: "banner",
-//         });
-//       }}
-//     />
-//   );
-// }
+function SendCTAEventToGA(eventCategory: string) {
```

Comment:
This function still starts with a capital letter, is that intentional?

```suggestion
function sendCTAEventToGA(eventCategory: string) {
```

Reply:
Yes, in order to use `useGA` it needs to follow the React naming convention.

Reply:
You will see that if you change it and run `yarn run eslint` you will get the following error:

```
React Hook "useGA" is called in function "sendCTAEventToGA" that is neither a React function component nor a custom React Hook function.
```

Reply:
Argh, I see!

Reply:
but does it actually work fine to use a React hook in a function that is called in an event listener ? That's not a supported way of using React hooks. The eslint rule is here to catch actual mistakes, not just about forcing a naming convention (it relies on the naming convention to detect the intended usage)

Reply:
You may be right @stof. I will take a look and remedy if needed. Thanks!